utilities/transactions/pessimistic_transaction_db.cc

```diff
@@ -683,30 +683,22 @@ void WritePreparedTxnDB::AdvanceMaxEvictedSeq(SequenceNumber& prev_max,
     }
   }
 
-  // With each change to max_evicted_seq_ fetch the live snapshots behind it
-  SequenceNumber curr_seq;
+  // With each change to max_evicted_seq_ fetch the live snapshots behind it.
+  // We use max as the version of snapshots to identify how fresh are the
+  // snapshot list. This works because the snapshots are between -infinity and
```

Comment:
by -infinity you mean 0?

Reply:
:) yes. I went too far in the negative axe.